barretenberg/cpp/src/barretenberg/goblin/goblin.hpp

```diff
@@ -88,7 +95,8 @@ class Goblin {
      * @return true
      * @return false
      */
-    static bool verify(const GoblinProof& proof);
+    static bool verify(const GoblinProof& proof,
```

Comment:
Would be nice if we could get rid of the default transcript arg, especially because it seems like its only needed in tests (I assume)